LICENSE

```diff
@@ -0,0 +1,14 @@
+Copyright 2018 Appnexus Inc.
```

Comment:
The copyright should be `PREBID.ORG, INC`

Reply:
OK, I'll do it.